test/ecies.js

```diff
@@ -132,4 +132,31 @@ describe('ECIES', function() {
     }).should.throw('Invalid checksum');
   });
 
+  it('decrypting compressed and uncompressed keys', function() {
+    var secret = 'test';
+
+    // test uncompressed
+    var alicePrivateKey = new bitcore.PrivateKey.fromObject(new Buffer('1fa76f9c799ca3a51e2c7c901d3ba8e24f6d870beccf8df56faf30120b38f360', 'hex'));
+    var alicePublicKey = new bitcore.PublicKey.fromPrivateKey(alicePrivateKey) // alicePrivateKey.publicKey
+    alicePrivateKey.compressed.should.equal(false)
+
+    var cypher1 = ECIES().privateKey(alicePrivateKey).publicKey(alicePublicKey);
+    var encrypted = cypher1.encrypt(secret);
+
+    var cypher2 = ECIES().privateKey(alicePrivateKey).publicKey(alicePublicKey);
+    var decrypted = cypher2.decrypt(encrypted);
+    secret.should.equal(decrypted.toString());
+
+    // test compressed
```

Comment:
This could be a new test e.g. `it('decrypting compressed keys` here. Also redefining of variables won't raise jshint warnings, this way.